### PR TITLE
Pass file arguments to gulp fix/validate

### PR DESCRIPTION
**Pull request by kimdomenick**
## Summary

- Install Yargs package for passing file params
- Update gulp validate/fix scripts and docs

## Metadata

| Question | Answer |
|----------|--------|
| Did you use a meaningful pull request title? | yes
| Did you apply meaningful labels to the pull request? | no
| Did you perform a self review first? | yes
| Documentation reflects changes? | yes
| `CHANGELOG` reflects changes? | yes
| Unit/Functional tests reflect changes? | no
| Did you perform browser testing? | no
| Risk level | Low
| Relevant links | JIRA issue, bug reports, security alerts, etc.
https://thinkoomph.jira.com/browse/BCBSALLY-1694

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -87,7 +88,7 @@ gulp
   })
   .task('validate:js', () => {
     return gulp
-      .src(javascriptFilePaths)
+      .src(fileArgs.file? fileArgs.file : javascriptFilePaths)
```
Comment (ben-hamelin) [inline gulpfile.js:91]: nit pick - can you add a space between "file" and "?" in these ternary statements?

**Comment by ben-hamelin** [comment]
@kimdomenick 
Just wondering why the package-lock.json is being introduced now? Seems appropriate to have it but I'm wondering if you know why it wasn't there prior.

Other than that just had the request on formatting those few lines.

File: gulpfile.js (hunk 1 above)
Comment (kimdomenick) [reply gulpfile.js]: Updated!

**Comment by kimdomenick** [comment]
@ben-hamelin - This was my first look at the scaffold project.  I committed the lock file, because it changed with the install of the yargs package.  Interesting that it wasn't there prior.

**Comment by ben-hamelin** [comment]
@kimdomenick - Ok this looks good! I've seen repos where we keep package-lock and others where we don't. Seems like we should lean towards the former in line with how we manage composer.lock. Thanks!
/approved

**Pull request merged**